digit-recognition/frontend/vite.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vite";
+import path from "path";
+import react from "@vitejs/plugin-react";
+import svgr from "vite-plugin-svgr";
+
+// https://vite.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "src"),
+    },
+  },
+  plugins: [react(), svgr()],
```

Comment:
Add `vite-plugin-checker` please to display TS and ESlint errors during development, otherwise a lot of things get missed.